src/test/config.sol

```diff
@@ -40,6 +40,7 @@ contract Config {
         uint256 vow_hump_max;
         uint256 split_hop;
         uint256 split_burn;
+        bytes32 split_farm;
```

Comment:
Shouldn't this be the actual address of the farm?
```suggestion
        address split_farm;
```

Reply:
See the other answer [above](https://github.com/makerdao/spells-mainnet/pull/433#discussion_r1803809853)